### DISCUSSION TITLE
Is this react native package still alive

### DISCUSSION THREAD

**Discussion by umardev500** [react-native-modal/react-native-modal#738] [General]
As you can see latest commit is in a years

**Comment by mmazzarolo** [comment]
https://github.com/react-native-modal/react-native-modal/discussions/598

**Comment by elencho** [comment]
it is not working on new architecture :( I wish I knew kotlin and swift, I would love to contribute

**Comment by hookFang** [comment]
Alternative can be found here -> https://github.com/gorhom/react-native-bottom-sheet
